### PR TITLE
MAINT: Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
User and non-user facing typos. 
some source typos fixes as well. Found via `codespell`

### CODE REVIEW

File: pavement.py [hunk 1]
```diff
@@ -303,7 +303,7 @@ def bootstrap(options):
     bdir = options.bootstrap_dir
     if not os.path.exists(bdir):
         os.makedirs(bdir)
-    bscript = "boostrap.py"
+    bscript = "bootstrap.py"
```
Comment (mattip) [inline pavement.py]: This changes the script name, which should make something that succeeded fail, or something that failed succeed

Comment (luzpaz) [reply pavement.py]: I was going to ask about this specific typo. Thanks for catching it. Deferring to the devs

**Comment by charris** [comment]
Most of the pavement file should be deleted, the referenced bit (bento related) among others. The small remaining bits could possibly be made a regular script.

**Comment by eric-wieser** [comment]
Thanks @luzpaz

**Pull request merged**